marconi-sidechain/src/Marconi/Sidechain/Run.hs

```diff
@@ -30,6 +31,9 @@ import Text.Pretty.Simple (pShowDarkBg)
 * marconi indexer workers
 
 Exceptions in either thread will end the program
+
+If the program is terminated with SIGINT or SIGTERM, exceptions in the marconi indexer workers
+thread will be mapped to exit codes
```

Comment:
It would be useful to add a link to the `signalToInt` function or to describe the mapping in the comment.